ctwrap/simulation.py

```diff
@@ -218,16 +217,15 @@ def save_metadata(output: Dict[str, Any],
             if oname is None:
                 return
             if opath is not None:
-                oname = os.path.join(opath, oname)
+                oname = PurePath.joinpath(Path(opath), oname)
 
             with h5py.File(oname, 'r+') as hdf:
                 for key, val in metadata.items():
                     if isinstance(val, dict):
-                        hdf.attrs[key] = yaml.dump(val, Dumper=yaml.SafeDumper)
+                        hdf.attrs[key] = json.dumps(val)
```

Comment:
I'm somewhat conflicted between YAML and JSON ... I'll likely have a closer look at how we add metadata in a different PR.

On a separate note, I just copied this function here as it's not used elsewhere.

Reply:
I reverted this in main: leave these lines unchanged ...